swim/node.go

```diff
@@ -58,7 +58,9 @@ type Options struct {
 func defaultOptions() *Options {
 	opts := &Options{
 		StateTimeouts: StateTimeouts{
-			Suspect: 5000 * time.Millisecond,
+			Suspect:   5 * time.Second,
+			Faulty:    1 * time.Hour,
```

Comment:
This seems indeed to aggressive. What are we trading off here?

- Large membership (low timeout) list versus.
- Time it takes to resolve a potential partition.

Unless there are more reasons why we want to keep this timeout low, I would keep this value to a few days.